wire.go

```diff
@@ -187,6 +187,10 @@ type StructFields struct{}
 //
 //  func NewStruct() *S { /* ... */ }
 //  var Set = wire.NewSet(wire.FieldsOf(new(*S), "MyFoo", "MyBar"))
+//
+//  If the structType argument is a pointer to a pointer to a struct, then FieldsOf
+//  additionally provides a pointer to each field type (e.g., **Foo and **Bar in the
```

Comment:
The example doesn't seem to be very good in this case. How about changing the fields of `S` to non-pointer type so that here you can say `*Foo` and `*Bar`?

Reply:
Done.